frontend/components/vault/VaultsText.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div class="flex flex-col space-y-8 py-8">
+        <WhatIsMakerProtocol v-if="isExplanationsShown" class="TextBlock"></WhatIsMakerProtocol>
+        <template v-if="isExplanationsShown">
+            <TextBlock title="What are vaults?" class="TextBlock">
+                A vault is a main source of DAI in the MakerDAO protocol: anyone is able to deposit supported
+                collateral into the system to get stablecoin in return and receive fees for the provided liquidity. The
+                place where the collateral is deposited is called the vault and is solely managed by the person who
+                deposited it, unless the deposited collateral amount no longer covers the withdrawn DAI.</TextBlock
```

Comment:
>and receive fees for the provided liquidity

This is not correct as the borrower doesn't receive fees but has to pay stability fees for the vault position. So I would change this half-sentence to: 
"and has to pay stability fees associated with the collateral type."